db/schema_tables.cc

```diff
@@ -87,10 +87,15 @@ static logging::logger diff_logger("schema_diff");
 namespace db {
```

Comment:
The change log should describe the change. Just "use_schema_commitlog" doesn't tell me much.

Reply:
The approach seems fine. You could post the static props refactoring separately, it's a nice cleanup.